OktaLogger.podspec

```diff
@@ -19,6 +19,7 @@ Pod::Spec.new do |s|
     complete.dependency 'OktaLogger/FileLogger'
     complete.dependency 'OktaLogger/FirebaseCrashlytics'
     complete.dependency 'OktaLogger/InstabugLogger'
+    complete.dependency 'OktaLogger/Analytics'
```

Comment:
It is already in the core impl from what I see in spec declaration

Reply:
cleaned up the code